documentation/docs/steps/newmanExecute.md

```diff
@@ -18,7 +21,30 @@ Step uses `dockerExecute` inside.
 
 ## Exceptions
 
-none
+The step has been migrated into a golang-based step. The following release notes belong to the new implementation:
+
+- **newmanRunCommand**:
+
+The parameter `newmanRunCommand` is deprecated by now and is replaced by list parameter `runOptions`. For backward compatibility, the `newmanRunCommand` parameter will still be used if configured. Nevertheless, using this parameter can break the step in some cases, e.g. when spaces are used in single quoted strings like spaces in file names. Also Groovy Templating is deprecated and now replaced by Go Templating. The example show the required changes:
+
+```yaml
+# deprecated groovy default
+newmanRunCommand: "run '${config.newmanCollection}' --environment '${config.newmanEnvironment}' --globals '${config.newmanGlobals}' --reporters junit,html --reporter-junit-export 'target/newman/TEST-${collectionDisplayName}.xml' --reporter-html-export 'target/newman/TEST-${collectionDisplayName}.html'"
+```
+
+```yaml
+# new run options using golang templating
+runOptions: ["run \{\{.NewmanCollection\}\}", "--environment", "\{\{.Config.NewmanEnvironment\}\}", "--globals", "\{\{.Config.NewmanGlobals\}\}", "--reporters", "junit,html", "--reporter-junit-export", "target/newman/TEST-\{\{.CollectionDisplayName\}\}.xml", "--reporter-html-export", "target/newman/TEST-\{\{.CollectionDisplayName\}\}.html"]
```

Comment:
How to escape (double) curly braces correctly? If they are not escaped, the docu generator crashes.

Reply:
Fixed.